diff --git a/CMakeLists.txt b/CMakeLists.txt
index 92e4698..4c2210f 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,6 +8,15 @@ cmake_minimum_required(VERSION 3.1)
 
 project(Ppconsul VERSION 0.1)
 
+### Conan integration
+include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
+conan_basic_setup()
+SET(BOOST_ROOT "${CONAN_BOOST_ROOT}")
+SET(CURL_ROOT "${CONAN_CURL_ROOT}")
+SET(Boost_NO_BOOST_CMAKE ON)
+SET(Boost_NO_SYSTEM_PATHS ON)
+##
+
 if (WIN32)
     option(BUILD_STATIC_LIB "Build Ppconsul as static library" ON)
 else()
